mainnet-deployment-info.json

```diff
@@ -4,14 +4,17 @@
   "treasury": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
   "matic_erc20_address": "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0",
   "matic_stake_manager_proxy": "0x5e3Ef299fDDf15eAa0432E6e66473ace8c13D908",
-  "proxy_admin": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
-  "maticX_proxy": "0xf03A7Eb46d01d9EcAA104558C732Cf82f6B6B645",
-  "maticX_impl": "0x648cf74330da142D4e774B14D6C79072c8Cc8701",
-  "validator_registry_proxy": "0xf556442D5B77A4B0252630E15d8BbE2160870d77",
-  "validator_registry_impl": "0x0a7f554AbFBD710D9CDFB7Cf88217a91acA7457D",
+  "eth_multisig": "0x80A43dd35382C4919991C5Bca7f46Dd24Fde4C67",
+  "eth_proxy_admin": "0x6CBd89A4919E39Ad4c7718B04443CC1722B2cB2A",
```

Comment:
eth_maticx_proxy_admin

Reply:
it is also the proxy admin of validator_registry

Reply:
it is also the proxy admin of validator_registry